packages/ui-mail-template/src/templates/layout-more-information.pug

```diff
@@ -1,4 +1,6 @@
 tr
-  td(style={ padding: '12px 24px' })
-    p
-      | More Information Here...
+  td(style= "padding: 12px 24px; text-align: center; color: #2B0444; font-size: 12px")
```

Comment:
Style should be converted to object to consistency with other components.

Let's re-check this PR @hiennq99 : https://github.com/xponvn/test-assessment/pull/80/files#diff-4f34fc05067584b22046b2339eabb86a41c3fea6f5597eb7edaf9e7c43980f5bR2

Reply:
fixed. thanks